src/shared/messages.js

```diff
@@ -20,12 +22,10 @@ const ERROR_GETTING_DATABASE_MESSAGE =
 const SUCCESS_LOADING_DATASET_MESSAGE = 'Dataset loaded successfully';
 const ERROR_LOADING_DATASET_MESSAGE = 'There was an error loading the dataset';
 const ERROR_MISSING_FILE_MESSAGE = 'The corresponding file was not found';
-const buildPythonProcessErrorMessage = (code) =>
-  `The Python process exited with code ${code}`;
-const ERROR_EXECUTING_PYTHON_ALGORITHM_MESSAGE =
-  'There was an error executing the Python algorithm';
-const SUCCESS_EXECUTING_PYTHON_ALGORITHM_MESSAGE =
-  'The Python algorithm was executed successfully';
+const buildProcessErrorMessage = (code) =>
+  `The process exited with code ${code}`;
+const SUCCESS_EXECUTING_ALGORITHM_MESSAGE =
+  'The execution was executed successfully';
```

Comment:
Now that there is `buidExecutingAlgorithmSuccessMessage`, is `SUCCESS_EXECUTING_ALGORITHM_MESSAGE` still needed?